notebooks/tel_and_site/subsys_req_ver/m1m3/LVV-T231-analysis-offline.ipynb

```diff
@@ -0,0 +1,169 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #41.            plt.plot(x_plot_mean, m*x_plot_mean + b, color='red', linewidth=2, label=f"Calculated Stiffness = {m:.2f} N/μm \nEncoder Fitting Range =  {round(x_near_zero.min(),2)} - {round(x_near_zero.max(),2)} μm")
</pre><p>I suggest you update this file to:</p><pre class="ql-syntax" spellcheck="false">plt.plot(
    x_plot_mean, 
    m * x_plot_mean + b, 
    color='red', 
    linewidth=2, 
    label=f"Calculated Stiffness = {m:.2f} N/μm\n"
          f"Encoder Fitting Range =&nbsp;{round(x_near_zero.min(),2)}"
          f" - {round(x_near_zero.max(),2)} μm"
    )
</pre><p>to comply with PEP-8, which says that code should be encapsuled within 100 characters per line. This also improves readability.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/notebooks_vandv/pull/46/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>